packages/react-core/src/components/JumpLinks/JumpLinks.tsx

```diff
@@ -91,18 +94,29 @@ export const JumpLinks: React.FunctionComponent<JumpLinksProps> = ({
   className,
   ...props
 }: JumpLinksProps) => {
-  const hasScrollSpy = Boolean(scrollableSelector);
+  const hasScrollSpy = Boolean(scrollableRef || scrollableSelector);
   const [scrollItems, setScrollItems] = React.useState(hasScrollSpy ? getScrollItems(children, []) : []);
   const [activeIndex, setActiveIndex] = React.useState(activeIndexProp);
   const [isExpanded, setIsExpanded] = React.useState(isExpandedProp);
   // Boolean to disable scroll listener from overriding active state of clicked jumplink
   const isLinkClicked = React.useRef(false);
-  // Allow expanding to be controlled for a niche use case
-  React.useEffect(() => setIsExpanded(isExpandedProp), [isExpandedProp]);
   const navRef = React.useRef<HTMLElement>();
 
   let scrollableElement: HTMLElement;
 
+  const getScrollableElement = () => {
+    if (scrollableRef) {
+      if ((scrollableRef as React.RefObject<HTMLElement>).current) {
+        return (scrollableRef as React.RefObject<HTMLElement>).current;
+      } else if (typeof scrollableRef === 'function') {
+        return scrollableRef();
+      }
+      return scrollableRef as HTMLElement;
```

Comment:
I think it's better if you first check for an `instanceof HTMLElement` so that you avoid the blind type cast for the react ref check.
```suggestion
      if (scrollableRef instanceof HTMLElement) {
        return scrollableRef;
      } else if (typeof scrollableRef === 'function') {
        return scrollableRef();
      }
      return (scrollableRef as React.RefObject<HTMLElement>).current;
```